js/bitfinex.js

```diff
@@ -398,38 +398,119 @@ module.exports = class bitfinex extends Exchange {
         };
     }
 
-    async fetchTradingFees (params = {}) {
+    async fetchTradingFee (symbol, params = {}) {
```

Comment:
Does this exchange have an endpoint that accepts a market id to return the fee information for just the specific market ? Cause if it does not have it, and we're not sending the market id to the exchange in the request, then we don't need this entire method, cause it's no better than `fetchTradingFees()`. The user can lookup a symbol in the dictionary returned from `fetchFundingFees` in the userland. Hence we don't need this method, if the exchange does not have a corresponding endpoint. Please verify that against their API docs.

Reply:
I'll remove it from this and other other bitfinex2 PR